### ISSUE TITLE
Building master fails at NetworkIntegrationTest

### ISSUE THREAD

**Issue by amccaskey** [jayjaybillings/fire#15]
Build fails in master if building from README instructions. Pulling down Fire and running cmake and make results in 

```cpp
In file included from /projects/test/fire/astrophysics/tests/NetworkIntegrationTest.cpp:40:0:
/projects/test/fire/solvers/ODESolver.h:35:36: fatal error: nvector/nvector_serial.h: No such file or directory
 #include <nvector/nvector_serial.h>
                                    ^
compilation terminated.
```

Are SUNDIALS/BLAS/LAPACK now required dependencies?

**Comment by jayjaybillings** [comment]
Hm. They are not required unless you want to build the astrophysics bundle. The solvers package will just skip them if they are not available.

I think we need to find a way to turn off bundles so that you are not trying to build the astrophysics bundle for your quantum work. Thoughts?

**Comment by amccaskey** [comment]
Yea I'd like a way to specifically turn on a bundle, and have it off otherwise. 

I noticed though there is no check in the astrophysics/CMakeLists file for the presence of those solver packages...

**Comment by jayjaybillings** [comment]
Well, yeah, I'm not done writing them yet. :-)

**Comment by amccaskey** [comment]
temporary fix for this in 885fff7d1a363f794dd53559e9f52819665129db

**Comment by jayjaybillings** [comment]
So, my fix was actually quite a bit different than yours. I just checked for the existence of the solvers libraries because the idea is that there will be something there, but maybe not sundials. In general we want to maintain encapsulation.
